src/streamlink/plugins/picarto.py

```diff
@@ -55,7 +55,7 @@ def _create_flash_stream(self, server, channel, token):
     def _get_vod_stream(self, page):
         m = self._vod_re.search(page.text)
         if m:
-            return HLSStream.parse_variant_playlist(self.session, m.group(1))
+            return HLSStream.parse_variant_playlist(self.session, m.group(1).replace('\\/', '/'))
```

Comment:
You could use `parse_json` if you included the `"` characters in the match group. It looks like JSON so there might be other things escaped for other streams...